LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Salil Cuncoliencar
```

Comment:
This is to be `ua-parser-extension` right?

Reply:
Yes, i have uodated it to ua-parser-extension

Reply:
Thanks for your time @salilbc and thanks for your review @reznord. 
I have already merged a PR for adding the license. However, as per @reznord's suggestion, it would be great if the name is the actual name of the package itself.

Reply:
If you have enough time, kindly pull, rebase and change the name to the actual name of the package and I'd love to merge your PR.